### ISSUE TITLE
Media keys not working when vscode is the active window

### ISSUE THREAD

**Issue by MLefebvreICO** [Microsoft/vscode#10223]
- **VSCode Version:** 1.4.0-insider
- **OS Version:** Windows 10 x64

**Steps to Reproduce:**

**1.** Use a keyboard that has macro support
**2.** Assign the media skip/back shortcut to one macro key

**What it should do:**
Change songs back and forth

**What it's currently doing:**
Nothing. Seems like `vscode` is capturing it, since it works everywhere else ...

Note: The "normal" media keys on the keyboard works properly, but not the macro I have.

**Comment by alexandrudima** [comment]
@MLefebvreICO Does the same happen to you when you press that specific media key in Chrome? What does the define keybinding widget detect when you press the media key?

File > Preferences > Keyboard Shortcuts -- ctrl+k ctrl+k:

![image](https://cloud.githubusercontent.com/assets/5047891/17486536/75427a1a-5d91-11e6-883b-4830ed06cc00.png)

**Comment by MLefebvreICO** [comment]
It does not happen in Chrome.

In vscode keyboard shortcuts, while in `Define Keybiding`, my macro doesn't seem to be logged as nothing shows up in the input.

However, if I press the "real" media keys of my keyboard it says : `unknown'.

Note that using the "real" media key it still change my song, but using my macro it doesn't.

The only other program that I have been using so far that does the same thing is `MINGW64` where my "real" media keys works, but not my macros.

**Comment by alexandrudima** [comment]
@MLefebvreICO AFAIK we don't do anything to keyboard events we don't recognize. i.e. we don't `preventDefault()` or try to capture them in any way. Given we have updated Electron recently, does this still reproduce for you? Is there a way to try to reproduce with a keyboard that does not support macros?

**Comment by alexandrudima** [comment]
I have just tested on Windows 10, using a keyboard that has Media buttons. I did not program the Media buttons. When I press the one labeled Mute while playing a video and having focus in VS Code, the Windows native UI for media controls pops up in the top left corner and indeed the computer is muted.

We don't do anything on our side to either prevent or disable media keys. Since there is no code change I can imagine us doing, I will close this issue at this time.
